packages/@sanity/cli/src/cli.js

```diff
@@ -69,6 +69,22 @@ module.exports = async function runCli(cliRoot) {
 }
 
 function getCoreModulePath(workDir) {
+  /* eslint-disable no-process-env */
+  if (process.env.SANITY_CLI_CORE_PATH) {
+    const local = fse.existsSync(process.env.SANITY_CLI_CORE_PATH)
+    if (local) {
+      console.log(
+        chalk.yellow(`[WARN] Loading @sanity/core from: ${process.env.SANITY_CLI_CORE_PATH}`)
+      )
+      return process.env.SANITY_CLI_CORE_PATH
+    }
+    console.warn(
```

Comment:
Also, if this is an error then perhaps it is best to use `output.error()` instead?

```suggestion
    output.error(```